deployment/terraform/firewall.tf

```diff
@@ -0,0 +1,55 @@
+#
+# Website ALB security group resources
+#
+resource "aws_security_group_rule" "raster_vision_alb_https_ingress" {
+  type        = "ingress"
+  from_port   = 443
+  to_port     = 443
+  protocol    = "tcp"
+  cidr_blocks = ["0.0.0.0/0"]
+
+  security_group_id = "${module.raster_vision_ecs_service.lb_security_group_id}"
+}
+
+resource "aws_security_group_rule" "alb_http_ingress" {
```

Comment:
Is this necessary if we're using CloudFront to handle HTTP -> HTTPS?

Reply:
Ah, it isn't.